presto-hive/src/test/java/io/prestosql/plugin/hive/benchmark/BenchmarkHiveFileFormat.java

```diff
@@ -504,44 +504,6 @@ public TestData createTestData(FileFormat format)
         return new TestData(columnNames, columnTypes, pages.build());
```

Comment:
change commit title. It now extracts `TestData` as top-level class

Reply:
good catch, fixed.